devicetypes/ezex/somfy-blind-controller-ezex.src/somfy-blind-controller-ezex.groovy

```diff
@@ -0,0 +1,123 @@
+
+metadata {
+    definition(name: "SOMFY Blind Controller/eZEX", namespace: "eZEX", author: "eZEX Corp", vid:"SmartThings-smartthings-Somfy_WindowShade", ocfDeviceType: "oic.d.blind") {
+        capability "Actuator"
+        capability "Configuration"
+        capability "Refresh"
+        capability "Window Shade"
+        capability "Health Check"
+        capability "Switch Level"
+
+        command "pause"
+
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0102", outClusters: "0019", model: "E2B0-KR000Z0-HA" // SY-IoT201-BD
+    }
+
+
+    tiles(scale: 2) {
+        multiAttributeTile(name:"windowShade", type: "generic", width: 6, height: 4) {
+            tileAttribute("device.windowShade", key: "PRIMARY_CONTROL") {
+                attributeState "open", label: 'Open', action: "close", icon: "http://www.ezex.co.kr/img/st/window_open.png", backgroundColor: "#00A0DC", nextState: "closing"
+                attributeState "closed", label: 'Closed', action: "open", icon: "http://www.ezex.co.kr/img/st/window_close.png", backgroundColor: "#ffffff", nextState: "opening"
+                attributeState "partially open", label: 'Partially open', action: "close", icon: "http://www.ezex.co.kr/img/st/window_open.png", backgroundColor: "#d45614", nextState: "closing"
+                attributeState "opening", label: 'Opening', action: "pause", icon: "http://www.ezex.co.kr/img/st/window_open.png", backgroundColor: "#00A0DC", nextState: "partially open"
+                attributeState "closing", label: 'Closing', action: "pause", icon: "http://www.ezex.co.kr/img/st/window_close.png", backgroundColor: "#ffffff", nextState: "partially open"
+            }
+        }
+        standardTile("contPause", "device.switch", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+            state "pause", label:"", icon:'st.sonos.pause-btn', action:'pause', backgroundColor:"#cccccc"
+        }
+        standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 1) {
+            state "default", label:"", action:"refresh.refresh", icon:"st.secondary.refresh"
+        }
+        valueTile("ShadeLevel", "device.level", width: 4, height: 1) {
+            state "level", label: 'Shade is ${currentValue}% up', defaultState: true
+        }
+        controlTile("levelSliderControl", "device.level", "slider", width:2, height: 1, inactiveLabel: false) {
+            state "level", action:"switch level.setLevel"
+        }
+
+        main "windowShade"
+        details(["windowShade", "contPause", "ShadeLevel", "levelSliderControl","refresh"])
+    }
+}
+
+private getCLUSTER_WINDOW_COVERING() { 0x0102 }
+private getATTRIBUTE_POSITION_LIFT() { 0x0008 }
+
+// Parse incoming device messages to generate events
+def parse(String description) {
+    def parseMap = zigbee.parseDescriptionAsMap(description)
+    log.info "LOG-INFO-Shade: In parse(), description:- ${description}"
```

Comment:
What do you see logged here? I ask because it looks like multiple attributes must be reported even though we only configured one for attribute reporting. In order to make sure you process all the attributes you need to do something like https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy#L98.

Reply:
fixed to log the only lift position lift attribute